src/pages/messages/TasksPage.tsx

```diff
@@ -162,7 +159,7 @@ const TasksPage = () => {
               }
             }}
             dataTestId={dataTestId.tasksPage.userDialogAccordion}>
-            <StyledTicketSearchFormGroup sx={{ gap: '0.5rem' }}>
+            <StyledTicketSearchFormGroup sx={{ gap: '0.5rem', mt: 0 }}>
```

Comment:
ALl filters are now pushed a little bit closer to the headline:

Before:
<img width="324" height="151" alt="image" src="https://github.com/user-attachments/assets/f04115f4-42a1-4f9c-9935-6597d2493282" />

Now:
<img width="320" height="143" alt="image" src="https://github.com/user-attachments/assets/ef5df813-49a7-4ec0-961a-54273e22c831" />